### DISCUSSION TITLE
My account has been blocked for no reason.

### DISCUSSION THREAD

**Discussion by eraoface0126** [vercel/community#4441] [Account Issue] [answered]
### Summary

I was trying to deploy my project to vercel.
However, when I clicked the “Continue with Github” button and verified, Vercel said that my account was blocked.
I didn't do anything.
Why did this happen?

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/146984424/46755534-82c4-42e7-baac-1c4ccb4eae19)

**Comment by srcnalt** [comment]
You might have received an email about the reason of block, skimming twitter I see some crypto/web3 users are getting blocked due to exhausting hobby account usage. Depending on your case it might be a breach of ToS maybe?

https://vercel.com/docs/limits/fair-use-guidelines

**Comment by eraoface0126** [comment]
I didn't receive any email. I didn't do anything. I just created an account using my github account.
Also my github account is new.

**Comment by amyegan** [comment, marked as answer]
I'm sorry you ran into this problem while creating a new account. We can't safely discuss account information in public because of security concerns.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
